### ISSUE TITLE
No such file or directory: 'ConEx//entity_to_idx.p'`

### ISSUE THREAD

**Issue by hzahera** [dice-group/dice-embeddings#80]
I downloaded the pre-trained ConEx from the Hobbit server using the command:
```
# To download a pretrained ConEx
mkdir ConEx && cd ConEx && wget -r -nd -np https://hobbitdata.informatik.uni-leipzig.de/KGE/DBpedia/ConEx/ && cd ..
```

I got this error with this code snippet: 
```
from core import KGE
pre_trained_kge = KGE(path_of_pretrained_model_dir='ConEx')
pre_trained_kge.predict_topk(head_entity=["http://dbpedia.org/resource/Albert_Einstein"],tail_entity=["http://dbpedia.org/resource/Ulm"])
```
Your help is appreciated 👍 

`FileNotFoundError    Traceback (most recent call last) Input In [9], in <cell line: 2>()       1 from core import KGE ----> 2 pre_trained_kge = KGE(path_of_pretrained_model_dir='ConEx')       3 pre_trained_kge.predict_topk(head_entity=["http://dbpedia.org/resource/Albert_Einstein"],tail_entity=["http://dbpedia.org/resource/Ulm"])  File /local/upb/users/z/zahera/profiles/unix/cs/dice-embeddings/core/knowledge_graph_embeddings.py:19, in KGE.__init__(self, path_of_pretrained_model_dir, construct_ensemble, model_name, apply_semantic_constraint)      17 def __init__(self, path_of_pretrained_model_dir, construct_ensemble=False, model_name=None,      18              apply_semantic_constraint=False): ---> 19     super().__init__(path_of_pretrained_model_dir, construct_ensemble=construct_ensemble, model_name=model_name,      20                      apply_semantic_constraint=apply_semantic_constraint)  File /local/upb/users/z/zahera/profiles/unix/cs/dice-embeddings/core/abstracts.py:142, in BaseInteractiveKGE.__init__(self, path_of_pretrained_model_dir, construct_ensemble, model_name, apply_semantic_constraint)     139         self.model, self.entity_to_idx, self.relation_to_idx = load_model(self.path + '/',     140                                                                           model_name=model_name)     141     else: --> 142         self.model, self.entity_to_idx, self.relation_to_idx = load_model(self.path + '/')     144 self.num_entities = len(self.entity_to_idx)     145 self.num_relations = len(self.relation_to_idx)  File /local/upb/users/z/zahera/profiles/unix/cs/dice-embeddings/core/static_funcs.py:89, in load_model(path_of_experiment_folder, model_name)      87 start_time = time.time()      88 print('Loading entity and relation indexes...', end=' ') ---> 89 with open(path_of_experiment_folder + '/entity_to_idx.p', 'rb') as f:      90     entity_to_idx = pickle.load(f)      91 with open(path_of_experiment_folder + '/relation_to_idx.p', 'rb') as f: 
 FileNotFoundError: [Errno 2] No such file or directory: 'ConEx//entity_to_idx.p'`

**Comment by Demirrr** [comment]
Yes this error is unfortunately expected since the model stored in Hobbit was trained with our old API. 

A work around would require creating a python dictionary of entity mappings `entity_to_idx.p` and relation mapings `relation_to_idx.p` 
from `entity_to_idx.gzip` and `relation_to_idx.gzip` respectively.
Please let me know if something is unclear.
